src/backend/main.ts

```diff
@@ -1141,6 +1141,56 @@ ipcMain.on('logError', (e, err) => logError(err, LogPrefix.Frontend))
 ipcMain.on('logInfo', (e, info) => logInfo(info, LogPrefix.Frontend))
 
 let powerDisplayId: number | null
+const gamePlaySessionStartTimes: Record<string, bigint> = {}
+
+function startNewPlaySession(appName: string) {
+  // Uses hrtime for monotonic timer not subject to clock drift or sync errors
+  const startPlayingTimeMonotonic = hrtime.bigint()
+  gamePlaySessionStartTimes[appName] = startPlayingTimeMonotonic
+}
+
+async function syncPlaySession(appName: string, runner: Runner) {
+  const stopPlayingTimeMonotonic = hrtime.bigint()
+  const sessionPlaytimeInMs =
+    (stopPlayingTimeMonotonic - gamePlaySessionStartTimes[appName]) /
+    BigInt(1000000)
+
+  // reset the time counter
+  startNewPlaySession(appName)
+
+  // update local json with time played
+  const sessionPlaytimeInMinutes =
+    sessionPlaytimeInMs / BigInt(1000) / BigInt(60)
+
+  const totalPlaytime =
+    sessionPlaytimeInMinutes + BigInt(tsStore.get(`${appName}.totalPlayed`, 0))
+  tsStore.set(`${appName}.totalPlayed`, Number(totalPlaytime))
+
+  const game = gameManagerMap[runner].getGameInfo(appName)
+  const { gameIsEpicForwarderOnHP, hyperPlayListing } =
+    await gameIsEpicForwarderOnHyperPlay(game)
+
+  if (gameIsEpicForwarderOnHP && hyperPlayListing) {
```

Comment:
Are we sending this for all Epic Forwarder games and all HyperPlay games even if they don't have a play session?

Reply:
well. they will always have a play session by this point. I think you're asking if the game doesn't have an active play streak quest and yes that's correct. We send this every time a game is played. 

When we add the quest id's to the listings endpoint, we can add this info to the `GameInfo` object and then check before calling this from the 'launch' handler.